### ISSUE TITLE
HGCN not quite right

### ISSUE THREAD

**Issue by raynamharris** [nih-cfde/update-content-registry#19]
In `ensemble_genes.tsv` the HGCN id is reported twice for any given gene. See TSPAN6 example:
- `[Source:HGNC Symbol;Acc:HGNC:11858]`
- `"HGNC:11858"`

```
ENSG00000000003	TSPAN6	tetraspanin 6 [Source:HGNC Symbol;Acc:HGNC:11858]	["ENTREZ:7105", "HGNC:11858", "MIM:300191", "NM_001278740", "NM_001278741", "NM_001278742", "NM_001278743", "NM_003270", "NP_001265669", "NP_001265670", "NP_001265671", "NP_001265672", "NP_003261", "T245", "TM4SF6", "TSPAN-6", "TSPAN6", "XM_011531018", "XP_011529320", "tetraspanin 6"]	NCBI:txid9606
```

The code to extract the HGNC id and create hyperlinks to the appropriate pages  is in `build-markdown-pieces-gene-translate.py`. The code look right, but the output is defaulting to value error of "None" instead of returning the ID, even though the ID is there for every gene.

![Screen Shot 2022-07-01 at 9 18 50 AM](https://user-images.githubusercontent.com/7544197/176933098-69d0ff96-350a-43cd-8271-595a4fa05cae.png)

(Note, in #15, I rotated the table so that the column names are now the row names, and I tried to directly call the URL instead of using the shortlink).

See

![176743882-ffb037d3-de83-47d4-a9e0-d99b2f16cbd0](https://user-images.githubusercontent.com/7544197/176933720-231c333e-d5bd-491f-bd3e-57c568a011ea.png)
![Screen Shot 2022-07-01 at 9 25 31 AM](https://user-images.githubusercontent.com/7544197/176933735-b71a2628-d6ce-4a65-8a24-2a12602bc78b.png)

**Comment by ctb** [comment]
I'll look into it now! thx!

**Comment by ctb** [comment]
I think I fixed it in 8c60d74, which I added to https://github.com/nih-cfde/update-content-registry/pull/15. 

<img width="889" alt="Screen Shot 2022-07-01 at 6 51 25 PM" src="https://user-images.githubusercontent.com/51016/176937365-d51c3425-875c-4cbe-ae9e-9eca0a8978bf.png">
